static/css/character_personality_onboarding.css

```
@@ -0,0 +1,616 @@
+    border-radius: 20px 20px 20px 4px;
+    background: var(--jelly-blue-50);
+}
+
+.character-personality-preview-stream {
+    position: relative;
+    color: var(--text-main);
+    font-size: 15px;
+    font-weight: 700;
+    line-height: 1.8;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)